stock_shipment_management/wizard/create_shipment.py

```diff
@@ -80,11 +80,9 @@ def _get_shipment_data(self, moves):
         if len(from_addresses) > 1:
             raise exceptions.Warning("Multiple From Address")
         to_addresses = moves.mapped('ship_to_address_id')
-        if len(to_addresses) > 1:
-            raise exceptions.Warning("Multiple To Address")
         data.update(
             from_address_id=from_addresses.id,
-            to_address_id=to_addresses.id
+            to_address_id=to_addresses[0].id
```

Comment:
As we said on IRC, I think this could crash if the field is empty, so we should handle that somewhat.